integration-tests/src/test/java/oracle/weblogic/kubernetes/extensions/ImageBuilders.java

```diff
@@ -582,5 +597,19 @@ private Callable<Boolean> pullImageFromOcrOrOcirAndPushToKind(String image) {
       return dockerPull(image) && dockerTag(image, kindRepoImage) && dockerPush(kindRepoImage);
     });
   }
+  
+  HelmParams opHelmParams;
+  String webhookNamespace;
+
+  private OperatorParams installWebHookOnlyOperator() {
+    webhookNamespace = assertDoesNotThrow(() -> createUniqueNamespace());
```

Comment:
Do we really need a unique namespace here? Or can it be set to fixed name such as `webhook-ns`. That will help when debugging the tests to identify the namespace associated with webhook.

Reply:
Recreating the webhook namespace before installing it. Using `ns-webhook` for namespace for installing webhook